rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.1.0-release-notes.adoc

```diff
@@ -7,15 +7,22 @@ RPA Recorder is MuleSoft’s tool for recording a business process for automatio
 // == What's New
 include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
 
-* The new attributes selection panel enables you to fine-tune the XPath for web elements.
-* You can now capture _Mouse Over_ events using the `F2` key.
-* You can now activate or deactivate hotkeys.
-* The new *Open log file* button enables you to open the latest log file.
-* The new *Open log files directory* enables you to open the folder containing all RPA Recorder logs in your local computer.
+* The new attributes selection panel enables you to fine-tune the XPath for web elements when you record Web Tasks.
+* You can now capture _Mouse Over_ events using the _F2_ key when you record Web Tasks.
+* You can now activate or deactivate hotkeys to control RPA Recorder.
+* The new *Log Files* tab in *Settings* contains buttons that enable easy access to log files and directories:
+** The new *Open log file* button enables you to open the latest log file.
+** The new *Open log files directory* enables you to open the folder containing all RPA Recorder logs in your local computer.
 * RPA Recorder changed the file name schema for all files written, introducing the following changes:
 ** The value `process name` changed to `process id`.
 ** Timestamps now appear at the beginning of file names instead of at the end.
-* RPA Recorder now adds the remote debugging port to the start arguments, which enables external access to the browser wizard.
+* Web Task recording now supports the following browser specific hotkeys to control browser navigation:
+** _F5_ : Performs a soft refresh of the page.
+** _Ctrl+F5_ : Performs a hard refresh of the page.
+** _Alt+Right arrow_ : Navigates forward.
+** _Alt+Left arrow_ : Navigates back.
+** _Ctrl+Enter_ : Adds `www.` and `.com` to the text in the URL bar and then navigates to this URL.
+// TBC - * RPA Recorder now adds the remote debugging port to the start arguments, which enables external access to the browser wizard.
```

Comment:
Checked with Alex and this can be removed, was only an internal, not customer facing change